client/src/components/useAuditAdminsOrganizations.ts

```diff
@@ -0,0 +1,15 @@
+import { useQuery, UseQueryResult } from 'react-query'
+import { fetchApi, ApiError } from '../utils/api'
+import { IAuditAdmin, IOrganization } from './UserContext'
+
+function useAuditAdminsOrganizations(
+  user: IAuditAdmin | null
+): UseQueryResult<IOrganization[], ApiError> {
+  return useQuery<IOrganization[], ApiError>(
+    'orgs',
+    () => fetchApi(`/api/audit_admins/${user?.id}/organizations`),
```

Comment:
nit: I see that the original code used to assert that the `user` is defined, e.g., `user!.id`, I presume because of the `{ enabled: !!user }` gate. Slight preference for the `!` over the `?` for clarity since user should never be undefined, unless I'm missing something